### DISCUSSION TITLE
add color gradient support to border/focus ring

### DISCUSSION THREAD

**Discussion by cptn-cosmo** [YaLTeR/niri#171] [Ideas]
please add an option to allow for color gradients in the border and focus ring configuration, like hyprland does in the default config

**Comment by YaLTeR** [comment]
Haven't looked at what Hyprland does but should be doable I think.

**Comment by jgregoire** [comment]
That would look incredibly slick.

**Comment by YaLTeR** [comment]
I added gradient borders in 48f0f6fb3ceb68fdb559ab38c8dcbb7b9ba3a13e.

**Comment by jgregoire** [comment]
<3

![image](https://github.com/YaLTeR/niri/assets/915427/afe8e2bd-52e6-4fa0-aadb-979bc9ee3433)
